ccm_web/client/src/models/FeatureUIData.tsx

```diff
@@ -76,5 +77,25 @@ const allFeatures: FeatureUIGroup[] = [
   { id: 'Sections', title: 'Sections', ordinality: 3, features: [mergeSectionCardProps, createSectionsCardProps] }
 ]
 
+const isAuthorizedForRoles = (userRoles: RoleEnum[], requiredRoles: RoleEnum[], featureDescriptionForLogging: string): boolean => {
+  const rolesAllowingAccess = userRoles.filter(requiredRole => requiredRoles.includes(requiredRole))
+  // These log messages seem useful while debugging/testing but should be removed later
+  if (rolesAllowingAccess.length > 0) {
+    console.debug('Feature ' + featureDescriptionForLogging + ' allowed by ' + rolesAllowingAccess.join(','))
+  } else {
+    console.debug('Feature ' + featureDescriptionForLogging + ' forbid for ' + userRoles.join(','))
+  }
+  return rolesAllowingAccess.length > 0
```

Comment:
Nit-pick: You could do `const isAuthorized = rolesAllowingAcess.length > 0`, and then reuse the variable here and in the logging check. This doesn't change anything, but might make the code a little tidier.